src/common/storage/Cargo.toml

```diff
@@ -8,6 +8,7 @@ edition = "2021"
 
 [features]
 storage-hdfs = ["opendal/services-hdfs"]
+storage-ftp = ["opendal/services-ftp"]
```

Comment:
Let's enable this by default. It's not very big and doesn't require external setup like `JAVA_RUNTIME`..